packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -28,6 +54,9 @@ const AppModals = ({ is_set_residence_modal_visible, url_action_param }) => {
     ) : null;
 };
 
-export default connect(({ ui }) => ({
-    is_set_residence_modal_visible: ui.is_set_residence_modal_visible,
+export default connect(({ client, ui }) => ({
+    is_set_residence_modal_visible    : ui.is_set_residence_modal_visible,
+    is_denial_of_service_modal_visible: !client.is_client_allowed_to_visit,
```

Comment:
Do we still need this considering the modal was removed?

Reply:
@aaron-binary the modal wasn't removed, it was moved from Trader to Core. This file is the one in Core. Am I misunderstanding what you meant?